src/data/W/instances.lean

```diff
@@ -88,35 +101,38 @@ def list_β : list_α γ → Type u
 
 instance (hd : γ) : inhabited (list_β γ (list_α.cons hd)) := ⟨ punit.star ⟩
 
-/-- The isomorphism from lists to W_types -/
-@[simp] def list_to_W_type : list γ → W_type (list_β γ)
+/-- The isomorphism from lists to the `W_type` construction of lists -/
+@[simp] def list_to : list γ → W_type (list_β γ)
```

Comment:
```suggestion
@[simp] def of_list : list γ → W_type (list_β γ)
```
`of_foo` and `to_foo` are usually the convention instead of `foo_to` and `to_foo`.

Reply:
done